pkg/loop/median_service_test.go

```diff
@@ -18,7 +18,39 @@ func TestMedianService(t *testing.T) {
 
```

Comment:
Let's use a table of subtests, rather than duplicating the body of the test:
```go
for _, tt := range []struct{
	name string
	gasPriceDataSource median.DataSource
} {
	{"static",  test.StaticGasPriceDataSource()},
	{"noop",  test.NOOPDataSource()},
} {
	t.Run(tt.name, func(t *testing.T) {
```

Reply:
thanks, good point